intg-androidtv/apps.py

```diff
@@ -63,4 +66,7 @@
     "plex": "Plex",
     "kodi": "Kodi",
     "emby": "Emby",
-}
+    "waipu": "Waipu TV",
+    "ard": "ARD Mediathek",
+    "zdf": "ZDF Mediathek",
```

Comment:
These name matching entries are not required, since they are already defined in the `IdMappings` above.
The `NameMatching` is a fallback match, where the application-id is not known, or if there are multiple ids.